### ISSUE TITLE
HTTP Basic Auth support?

### ISSUE THREAD

**Issue by markphelps** [samsonmking/epaper.js#59]
👋🏻  What do you think about adding HTTP Basic Auth support for `url`? 

My use case:

I want to host my page on the web so that I can make use of some Google APIs which require user interaction initially (calendar read events), however, I don't want to publically expose my data to the world. 

It would be great if I could use HTTP basic auth to protect my page on the server, and then pass those values through `ejs` via command line option like `--basic-auth`.

It seems puppeteer supports this like 

```
page.authenticate({'username':'YOUR_BASIC_AUTH_USERNAME', 'password': 'YOUR_BASIC_AUTH_PASSWORD'});
```

I think this could be added [here](https://github.com/samsonmking/epaper.js/blob/59e65bd0c149295876ca4dd87ace849a9c9f15c5/packages/core/src/page/page.ts#L13) and set with the values provided via `--basic-auth` to `ejs`

**Comment by samsonmking** [comment]
Hi @markphelps, good idea! Seems pretty straight forward, I'll take a look over the next few days.

**Comment by samsonmking** [comment]
@markphelps I've released this functionality in `@epaperjs/cli@2.1.0`! Please let me know if you run into any issues.

**Comment by markphelps** [comment]
Thanks @samsonmking !
